benchmarking/nursery/test_bolimit/requirements.txt

```diff
@@ -0,0 +1,2 @@
+syne-tune[gpsearchers,blackbox-repository,aws]
+tqdm
```

Comment:
where is the dependency `tqdm` coming from?

Reply:
`tqdm` is not among the Syne Tune dependencies, but it is needed for benchmarking.